tests/components/test_classifiers.py

```diff
@@ -8,14 +8,16 @@
     creates_mock_dataset, delete_mock_dataset
 from projects.controllers.utils import uuid_alpha
 
+import os
```

Comment:
Existe um padrão de ordem de imports:
https://www.python.org/dev/peps/pep-0008/#imports

1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.

`import os`  é da Standard library, então deve ficar no início dos imports